### PR TITLE
Executive Vote - December 9, 2022

### PR DESCRIPTION
**Pull request by astronautthis**
@prose11 @patrick-j-govalpha

### CODE REVIEW

File: governance/votes/Executive Vote - December 9, 2022.md [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+- Reduce the YFI-A Target Available Debt (`gap`) by 5.5 million DAI from 7 million DAI to **1.5 million DAI**.
+
+##### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Increases
+- Increase the WBTC-A Ceiling Increase Cooldown (`ttl`) by 18 hours from 6 hours to **24 hours**.
+- Increase the WBTC-B Ceiling Increase Cooldown (`ttl`) by 16 hours from 8 hours to **24 hours**.
+- Increase the WBTC-C Ceiling Increase Cooldown (`ttl`) by 16 hours from 8 hours to **24 hours**.
+
+#### [Peg Stability Module (PSM)](https://manual.makerdao.com/module-index/module-psm) Parameter Changes
+
+##### [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) Increases
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - December 9, 2022.md:287]: Move PSM line change to PSM section (this was my error in the original poll!
```suggestion

##### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Reductions
- Reduce the WSTETH-A Target Available Debt (`gap`) by 15 million DAI from 30 million DAI to **15 million DAI**.
- Reduce the WSTETH-B Target Available Debt (`gap`) by 15 million DAI from 30 million DAI to **15 million DAI**.
- Reduce the WBTC-A Target Available Debt (`gap`) by 60 million DAI from 80 million DAI to **20 million DAI**.
- Reduce the WBTC-B Target Available Debt (`gap`) by 20 million DAI from 30 million DAI to **10 million DAI**.
- Reduce the WBTC-C Target Available Debt (`gap`) by 80 million DAI from 100 million DAI to **20 million DAI**.
- Reduce the LINK-A Target Available Debt (`gap`) by 4.5 million DAI from 7 million DAI to **2.5 million DAI**.
- Reduce the YFI-A Target Available Debt (`gap`) by 5.5 million DAI from 7 million DAI to **1.5 million DAI**.

##### [Ceiling Increase Cooldown (`ttl`)](https://manual.makerdao.com/module-index/module-dciam#ceiling-increase-cooldown-ttl) Increases
- Increase the WBTC-A Ceiling Increase Cooldown (`ttl`) by 18 hours from 6 hours to **24 hours**.
- Increase the WBTC-B Ceiling Increase Cooldown (`ttl`) by 16 hours from 8 hours to **24 hours**.
- Increase the WBTC-C Ceiling Increase Cooldown (`ttl`) by 16 hours from 8 hours to **24 hours**.

#### [Peg Stability Module (PSM)](https://manual.makerdao.com/module-index/module-psm) Parameter Changes

##### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Reduction
- Reduce the PSM-USDP-A Maximum Debt Ceiling (`line`) by 50 million DAI from 500 million DAI to **450 million DAI**.

##### [Fee In (`tin`)](https://manual.makerdao.com/module-index/module-psm#fee-in-tin) Increases
```

File: governance/votes/Executive Vote - December 9, 2022.md [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+Please review the [discussion thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-24-november-2022/18925) containing information about the MOMC Proposal to inform your position before voting.
+
+### DAI Savings Rate Adjustment
+
+As per this [governance poll](https://vote.makerdao.com/polling/914), the following changes will be made to the Maker Protocol, if this executive proposal passes.
+- Increase the [DAI Savings Rate](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate) from 0.01% to **1%**.
+
+Please review the [discussion thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-24-november-2022/18925#dai-savings-rate-adjustment-3) containing information about the DAI Savings Rate Adjustment to inform your position before voting.
+
+## Review
```
Comment (patrick-j-govalpha) [inline governance/votes/Executive Vote - December 9, 2022.md:304]: Add DSR MOM link

```suggestion
Please review the [discussion thread](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-24-november-2022/18925#dai-savings-rate-adjustment-3) containing information about the DAI Savings Rate Adjustment to inform your position before voting.

For more information on the DAI Savings Rate please refer to the documentation [here](https://manual.makerdao.com/parameter-index/core/param-dai-savings-rate).

## Review
```

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]
Initial review - waiting for final Blocktower details so still need to review those sections

**Comment by patrick-j-govalpha** [comment]
@prose11 would appreciate your set of eyes on this too

**Review by patrick-j-govalpha — CHANGES_REQUESTED** [review]

File: governance/votes/Executive Vote - December 9, 2022.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+
+#### Relevant Smart Contract Information 
+
+* **SYMBOL:** `RWA011`.
+* **NAME:** `RWA-011`.
+* **ILK:** `RWA011-A`.
+* **RWA011:** `0x0b126F85285d1786F52FC911AfFaaf0d9253e37a`.
+* **MCD_JOIN_RWA011_A**: `0x9048cb84F46e94Ff312DcC50f131191c399D9bC3`.
+* **RWA011_A_URN:** `0x32C9bBA0841F2557C10d3f0d30092f138251aFE6`.
+* **RWA011_A_JAR:** ``.
```
Comment (miguel567) [inline governance/votes/Executive Vote - December 9, 2022.md:156]: Centrifuge vaults don't have Jar contracts. You can remove this line totally.

**Review by patrick-j-govalpha — APPROVED** [review]
Thanks everyone, huge copy this week, spare a thought for the poor devs

**Pull request merged**